project/RunSbtCommand.scala

```diff
@@ -1,28 +0,0 @@
-import sbt._
```

Comment:
As far as I know, this code is not used now.

Reply:
This is indeed no longer used after the sbt plugin was moved to a separate repo.